assets/src/components/TagSelector/TagSelector.js

```diff
@@ -1,63 +1,63 @@
-import { Component } from '@wordpress/element';
-import { compose } from '@wordpress/compose';
-import { withSelect } from "@wordpress/data";
+import {Component} from '@wordpress/element';
+import {compose} from '@wordpress/compose';
+import {withSelect} from '@wordpress/data';
 import {
   FormTokenField,
 } from '@wordpress/components';
 
 class TagSelector extends Component {
-  constructor( props ) {
-    super( props );
+  constructor(props) {
+    super(props);
 
-    this.handleChange = this.handleChange.bind( this );
-    this.getValue = this.getValue.bind( this );
+    this.handleChange = this.handleChange.bind(this);
+    this.getValue = this.getValue.bind(this);
   }
 
-  handleChange( value ) {
-    const tagIds = value.map( token => {
-      return this.props.tagSuggestions.find( tag => tag.name === token ).id;
+  handleChange(value) {
+    const tagIds = value.map((token) => {
+      return this.props.tagSuggestions.find((tag) => tag.name === token).id;
     });
-    this.props.onChange( tagIds );
+    this.props.onChange(tagIds);
   }
 
   getValue() {
-    const { tagSuggestions, value } = this.props;
+    const {tagSuggestions, value} = this.props;
 
-    if ( !tagSuggestions || !value ) {
+    if (!tagSuggestions || !value) {
       return null;
     }
 
-    const tags = value.reduce( ( accumulator, tagId ) => {
-      const tag = tagSuggestions.find( tag => Number( tag.id ) === Number( tagId ) );
-      if ( tag ) {
-        accumulator.push( tag );
+    const tags = value.reduce((accumulator, tagId) => {
+      const tag = tagSuggestions.find((tagFound) => Number(tagFound.id) === Number(tagId));
+      if (tag) {
+        accumulator.push(tag);
       }
       return accumulator;
-    }, [] );
+    }, []);
 
-    return tags.map( tag => tag.name );
+    return tags.map((tag) => tag.name);
   }
 
   render() {
-    const { tagSuggestions, onChange, label, placeholder, value, ...ownProps } = this.props;
+    const {tagSuggestions, label, placeholder, ...ownProps} = this.props;
```

Comment:
We actually need to keep `onChange` and `value` here, even if unused, otherwise they are in `ownProps` and when we later pass it down, they override the ones what we set (I hope that makes sense? 😅)

Reply:
Oh my bad, got it 👍